pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -167,6 +167,29 @@ class Foo {
         </example>
     </rule>
 
+    <rule name="AvoidReassigningLoopVariables"
+          since="6.11.0"
+          message="Avoid reassigning the loop control variable ''{0}''"
+          class="net.sourceforge.pmd.lang.java.rule.bestpractices.AvoidReassigningLoopVariablesRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_bestpractices.html#avoidreassigningloopvariables">
+        <description>
+Reassigning loop control variables can lead to hard-to-find bugs. Prevent or limit how these variables can be changed.
```

Comment:
The possible values for your properties, how their affect the rule, and their defaults should be documented here. Possibly also add comments to your example code to show how different options would cause a different result.
We'll improve how properties are documented as part of #1432, but for now the rule description is the best way to do that

Reply:
I documented the properties/their possible values, including a description/explanation for the intermediate options.